controllers/api-controllers.js

```diff
@@ -0,0 +1,10 @@
+const {readApiDetails} = require("../models/api-models")
+
+exports.getApiDetails = (req, res, next) => {
+    readApiDetails().then((details) => {
+        res.status(200).send({api: details})
```

Comment:
Rather than having a model and invoking it here, we can simply require in the endpoints.json file and send it from here 👍